File: app/src/main/java/com/inlustris/cuccina/Beans/Ingredient.java
package com.inlustris.cuccina.Beans;

public class Ingredient {
    String count;

    public String getCount() {
        return count;
    }

    public void setCount(String count) {
        this.count = count;
    }

    String ingrediente;
    String medidas;
    String quantidade;




    public Ingredient() {
    }

    public Ingredient(String count, String ingrediente, String medidas, String quantidade) {
        this.count = count;
        this.ingrediente = ingrediente;
        this.medidas = medidas;
        this.quantidade = quantidade;
    }

    public String getIngrediente() {
        return ingrediente;
    }

    public void setIngrediente(String ingrediente) {
        this.ingrediente = ingrediente;
    }

    public String getMedidas() {
        return medidas;
    }

    public void setMedidas(String medidas) {
        this.medidas = medidas;
    }

    public String getQuantidade() {
        return quantidade;
    }

    public void setQuantidade(String quantidade) {
        this.quantidade = quantidade;
    }
}

File: app/src/main/java/com/inlustris/cuccina/beans/Ingredient.kt
package com.inlustris.cuccina.beans


class Ingredient {
    var count: String? = null


    var ingrediente: String? = null
    var medidas: String? = null
    var quantidade: String? = null

    constructor()
    constructor(count: String?, ingrediente: String?, medidas: String?, quantidade: String?) {
        this.count = count
        this.ingrediente = ingrediente
        this.medidas = medidas
        this.quantidade = quantidade
    }

}